src/constants.js

```diff
@@ -21,6 +21,8 @@ export const TestDataLocations = {
   RANDOM: "random"
 };
 
+export const ModelNameMaxLength = 256;
```

Comment:
Should we make it smaller just because anything above something like (say) 50, feels like it might be a mistake?